tests/test_modeling_common.py

```diff
@@ -1284,3 +1288,9 @@ def test_model_from_pretrained(self):
             model = BertModel.from_pretrained(model_name, output_attentions=True, output_hidden_states=True)
             self.assertEqual(model.config.output_hidden_states, True)
             self.assertEqual(model.config, config)
+
+    def test_model_from_pretrained_with_different_pretrained_model_name(self):
+        model = BertModel.from_pretrained(TINY_BERT)
```

Comment:
if you were to test T5 against t5 then you don't need to add `TINY_BERT = "lysandre/tiny-bert-random"` - one model is enough here - use 2 classes instead.

Reply:
Hi,
I don't understand the last part about using two classes. 
Should I add a separate class for this particular test?

Reply:
The change you applied is it. I was trying to communicate that the test needs only one pretrained model and one good class and one bad class = 2 classes.